.github/workflows/public-ci.yml

```diff
@@ -39,5 +39,5 @@ jobs:
           EMOJI: ${{ needs.test.result == 'success' && ':party-hat:' || ':boom:' }}
           STATUS: ${{ needs.test.result == 'success' && 'succeeded!' || 'failed.' }}
         run: |
-          PAYLOAD='{"text":"${{ env.EMOJI }} CI testing of ${{ github.event.repository.full_name }} has ${{ env.STATUS }}\nCommit <${{ github.event.commits[0].url }}|${{ github.sha }}> by ${{ github.event.commits[0].author.name }}: ${{ github.event.commits[0].message }}\n<https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|Test results>"}'
+          PAYLOAD="{\"text\":\"${{ env.EMOJI }} CI testing of ${{ github.event.repository.full_name }} has ${{ env.STATUS }}\\nCommit <${{ github.event.commits[0].url }}|${{ github.sha }}> by ${{ github.event.commits[0].author.name }}: ${{ github.event.commits[0].message }}\\n<https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|Test results>\"}"
```

Comment:
Slightly confused. If I use double quotes in the commit message, what will happen? I.e. what does this fix *exactly*?

Reply:
Just a FYI that the "auto-remove" means that the string ends and it happens to work because there is no white space in `test` and therefore between the ending and starting double quote. I doubt it will work as expected in longer quoted strings when you e.g. quote `"This is a test"` and use that within a larger comment. Just a FYI.